test/performance/vectorize_pred.cpp

```diff
@@ -1,45 +1,68 @@
 #include "Halide.h"
-#include <cstdio>
 #include "halide_benchmark.h"
+#include <cstdio>
 
 using namespace Halide;
 using namespace Halide::Tools;
 
 template<typename A>
 const char *string_of_type();
 
-#define DECL_SOT(name)                                          \
```

Comment:
This should just go away and the printing should be done using ```Halide::Type``` and the stream ```operator<<``` from IRPrinter.

Reply:
+1. (I think this is ready to land aside from this nit.)

Reply:
done 👍